lib/deck.rb

```diff
@@ -0,0 +1,14 @@
+class Deck
+
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
```

Comment:
The interaction pattern indicates we want to create a method called `count` for `Deck` specifically. Something like this should do the trick:
```ruby
def count
  @cards.count
end
```

It does feel a little wonky to have this since it's not doing a whole lot for us, but it can save us a step later on by including it.